### PR TITLE
Decaffeinate

### PR DESCRIPTION
**Pull request by natestemen**
### Description
Decaffeinate this repo.

#### Related Issues / PRs
https://github.com/overleaf/web-internal/issues/2555

#### Manual Testing Performed

- [x] created a new tag
- [x] deleted an old tag
- [x] added a tag to a project
- [x] filtered the project list by a tag
- [x] created a new tag via multiselect
- [x] removed tags via multiselect
- [x] can see v1 tags

### CODE REVIEW

File: test/acceptance/js/CRUDTests.js [hunk 1]
```diff
@@ -0,0 +1,335 @@
...
+const {ObjectId} = mongojs;
+
+const Settings = require('settings-sharelatex');
+const db = mongojs(Settings.mongo.url, ['tags']);
+
+const TagsApp = require('./helpers/TagsApp');
+const TagsClient = require('./helpers/TagsClient');
+
+describe('CRUDTests', function() {
+	before(function(done) { return db.tags.createIndex({
```
Comment (JuneKelly) [inline test/acceptance/js/CRUDTests.js]: I notice that we use tabs for indent in these test files. Could we change this to spaces? Or even better, will `prettier` change it for us?

**Review by JuneKelly — COMMENTED** [review]
I think this looks good :tada: 

I've left one comment on tabs/spaces. One more question would be: given the small size of this repo, would it make sense to go through the main files and do the decaf-cleanup step now? Otherwise I think we'll never get around to it, as we don't touch this service very often.

File: test/acceptance/js/CRUDTests.js (hunk 1 above)
Comment (natestemen) [reply test/acceptance/js/CRUDTests.js]: Prettier is configured (by default) to indent using spaces. Somehow this file never got hit by eslint. I've put together a pr to fix this https://github.com/overleaf/dev-environment/pull/308. Once it is merged I will run it again and force push.

**Comment by chrystalgriffiths** [comment]
Seems to be working well and looks fine, just needs conflicts sorting out

**Review by JuneKelly — APPROVED** [review]
Looks good :tada:

**Pull request merged**